src/main/resources/feedbackQuestionConstSumEditFormOptionFragment.html

```diff
@@ -1,5 +1,5 @@
-<div id="constSumOptionRow-${i}-${questionNumber}">
-    <div class="input-group col-sm-12">
+<div class="margin-bottom-7px" id="constSumOptionRow-${i}-${questionNumber}">
+    <div class="input-group width-100-pc">
```

Comment:
even this is a bad name, but I think someone has already defined this earlier. See if you can change this (if its usage is limited within this pr)